Add speaker's dinner page (#606)
EuroPython/website: src/content/pages/speakers-dinner.mdx

---
title: Speakers' Dinner
url: /speakers-dinner
subtitle:
  We're happy to invite all speakers to our special Speakers' Dinner!
---

> # Speaker's Dinner

Reviewer: ```suggestion
# Speakers' Dinner
```
Just to be consistent throughout.

We are excited to announce a special Speaker's Dinner! This exclusive event is for speakers only, providing an opportunity to meet and mingle before the main conference days.

> The dinner will feature multiple courses, and you can choose from a variety of alcoholic and non-alcoholic beverages. We also offer vegetarian, vegan, and gluten-free options to accommodate all dietary preferences.

Reviewer 2: ```suggestion
The dinner will feature multiple courses and you can choose from a variety of alcoholic and non-alcoholic beverages. We also offer vegetarian, vegan, and gluten-free options to accommodate all dietary preferences.
```
I believe that food and drinks should be kept together and therefore the separating "," is not the right choice?

Author: ChatGPT says `,` makes it more readable. 🤷🏻‍♀️ 

![image](https://github.com/EuroPython/website/assets/38294198/c55a4d90-428b-4d5a-87a6-12bfa56dca6c)

Reviewer: > vegetarian, vegan, and gluten-free options to accommodate all dietary preferences.

this sentence suggests vegetarian, vegan and gf are all the dietary preferences. 

If you don't have any other option, perhaps something like this is clearer: "We also offer  vegetarian, vegan, and gluten-free options to cater to various dietary preferences."

If you have other options to choose from, maybe "We also offer a variety of options to accommodate different dietary preferences, including vegetarian, vegan, and gluten-free."?

Author: Thanks for the suggestion, I've included the second sentence.

# Where?

> The Speakers' Dinner will be at the Mánes restaurant on the Vltava riverfront in Prague, featuring creative functionalist architecture and panoramic views of the Vltava River, the Dancing House, and the National Theatre.

Reviewer: > at the Mánes restaurant
Might it be a good idea to link the URL to the restaurant?

Author: added ✅

Address:
<address>
Masarykovo nábř. 250/1 <br/>
110 00 Nové Město, Czechia<br/>

</address>

![](./images/manes.jpg)

> ## How to arrive there?

Reviewer 2: ```suggestion
## How to get there?
```
Probably nitpicking.

> * Take trams 2, 5, 9, 13, 15, 17, 18, 19, 22, 24, or 27 to Jiráskovo náměstí station, which is a 2-minute walk from the restaurant.

Reviewer 2: ```suggestion
* Take trams 2, 5, 9, 13, 15, 17, 18, 19, 22, 24, or 27 to Jiráskovo náměstí station. From there it is a 2-minute walk to the restaurant.
```

* [30 minutes on foot](https://www.google.com/maps/dir/Prague+Congress+Centre,+5.+kv%C4%9Btna,+Prague+4-Nusle,+Czechia/Art+Restaurant+M%C3%A1nes,+Masarykovo+n%C3%A1b%C5%99.+250%2F1,+110+00+Nov%C3%A9+M%C4%9Bsto,+Czechia/@50.0697187,14.4116608,15z/data=!4m15!4m14!1m5!1m1!1s0x470b9464c186eb79:0x4d26855708eb61f7!2m2!1d14.4290206!2d50.0625764!1m5!1m1!1s0x470b952eddf2f267:0xbd409c6e6af026e!2m2!1d14.41412!2d50.0774996!3e2!5i2?entry=ttu) from the Prague Congress Center

# When?
On Tuesday July 9th at 7.30 p.m.

# Registration

> Please note that registration is required. Register through Pretix by adding the event to your list.

Reviewer 2: ```suggestion
Please note that registration is required. Register through Pretix by adding the event to your speaker ticket.
```
